covid19-dashboard/main.py

```diff
@@ -498,5 +487,13 @@ def add_browser_cache_headers_to_response(data: Dict,
     return response
 
 
+def _place_info_factory(name, contained_in, place_type):
```

Comment:
this is technically not a factory... why not just call it _create_place_info.
also probably worth making it a type.

Reply:
i made it into a class and then do __dict__. is  that ok?